### PR TITLE
[W5.11][W11-A1]Lai Qi Wei

### PR DESCRIPTION
**Pull request by qwlai**
Added sort command to sort names by alphabetical order. 
Updated tests and JUnit tests for sort command.
Updated the relevant documentations as well.

### CODE REVIEW

**Comment by wzneo** [comment]
Great job! Perhaps, the tests can be better targeted to showcase the sorting in effect.

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 1]
```diff
@@ -72,6 +72,13 @@ public UniquePersonList(UniquePersonList source) {
         internalList.addAll(source.internalList);
     }
 
+    /**
+     * Sorts all persons in the address book by name in alphabetical order.
+     */
+    public void sortByName() {
+        internalList.sort((person1, person2) -> person1.getName().toString().compareToIgnoreCase(person2.getName().toString()));
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/UniquePersonList.java:79]: Good use of lambdas :+1:

File: test/expected.txt [hunk 2]
```diff
@@ -135,9 +137,32 @@
 || 
 || 5 persons listed!
 || ===================================================
+|| Enter command: || [Command entered:  add Alex p/666666 e/alex@gmail.com a/111, falcon street t/work]
+|| New person added: Alex Phone: 666666 Email: alex@gmail.com Address: 111, falcon street Tags: [work]
+|| ===================================================
+|| Enter command: || [Command entered:  list]
+|| 	1. Adam Brown Phone: 111111 Email: adam@gmail.com Address: 111, alpha street Tags:
```
Comment (okkhoy) [inline test/expected.txt:144]: nice test case showing the before and after of `sort`

File: test/java/seedu/addressbook/commands/SortCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.util.TestUtil;
+
+public class SortCommandTest {
```
Comment (okkhoy) [inline test/java/seedu/addressbook/commands/SortCommandTest.java:21]: Can consider adding header comments here as well.

**Review by okkhoy — COMMENTED** [review]
Good job on a clean PR :+1:

**Comment by okkhoy** [comment]
@qwlai 
Some comments added. Please ack & close the PR after reading comments.

**Comment by qwlai** [comment]
Noted with thanks.

**Pull request closed without merging**